Game/Scripts/automation.py

```diff
@@ -0,0 +1,19 @@
+def onPGV(args):

```

Comment:
I'd like a more descriptive name for this function without an abbreviation.

Reply:
Created a new pull request with this fixed.